src/main/java/stream/flarebot/flarebot/commands/general/ReportCommand.java

```diff
@@ -50,6 +52,8 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
 
             guild.getReportManager().report(report);
 
+            guild.getAutoModConfig().postToModLog(user, sender, new Punishment(ModlogAction.REPORT_CREATE), MessageUtils.getMessage(args, 1));
```

Comment:
This shows the reportee as the "responsible moderator" may be best to use the EmbedBuilder specific way of doing this.

Reply:
I changed it. take a look and see what you think